### DISCUSSION TITLE
Favorited article deleted after rss feed removed

### DISCUSSION THREAD

**Discussion by zapatistay** [FreshRSS/FreshRSS#5046] [Q&A]
I just removed an rss feed, and noticed that an articles I had favourited belonging to that rss feed is gone. Is this intentional or a bug? So all the favourited articles belonging to an rss feed will be deleted after unsubscription to that feed?

I would expect a favourited article to be deleted/removed if and only if I "unstar" it, not as an indirect consequence of some action taken in another page.

**Comment by Alkarex** [comment]
Indeed, this is the current behaviour: articles cannot exist without their feed. So instead of deleting feeds, it is possible to *mute* a feed and it will not be refreshed nor be visible in the main view.
However in the future, it would indeed be good to have another way to keep favourites and labelled articles (we might have an issue on it somewhere).

**Reply by zapatistay** [reply]
By mute (or pause), are you referring to the Visibility pull-down list ?

**Comment by Frenzie** [comment]
There's also an issue (or only a mention) that we should really call "mute" pause instead of mute, but I think I forgot about it. :-)
